#!/bin/sh
xvfb-run python3 main.py > err.log